### PR TITLE
Fred/readme 3

### PR DESCRIPTION
**Pull request by flintzPax8**
do it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ## you call this a readme bro?
 
-### yes i do
\ No newline at end of file
+### yes i do
+
```
Comment (flintzPax8) [inline README.md:4]: commit comment please

**Review by fredlintz5 — APPROVED** [review]
Hahahah, I approve of this pr.

**Review by fredlintz5 — CHANGES_REQUESTED** [review]
requested changes

**Review by sfrendpax8 — COMMENTED** [review]
qwetqwetqwte

**Pull request closed without merging**